tests/lib/nm/ovs_test.py

```diff
@@ -122,8 +122,6 @@ def test_get_ovs_info_with_ports_with_interfaces(
 
     assert len(info["port"]) == 1
     assert "name" in info["port"][0]
-    assert "vlan-mode" in info["port"][0]
```

Comment:
These removals fit a separate commit/PR. I think you cleaned the implementation as well.